bazel/repository_locations.bzl

```diff
@@ -76,8 +76,8 @@ REPOSITORY_LOCATIONS = dict(
         urls = ["https://github.com/google/protobuf/archive/v3.5.0.tar.gz"],
     ),
     envoy_api = dict(
-        commit = "92ffa417b5d2e8ab3912ff4da4daa6aa93f8df0f",
-        remote = "https://github.com/envoyproxy/data-plane-api",
+        commit = "55f78650483140f8dbce89d6b215e85d52aaefba",
```

Comment:
I just merged https://github.com/envoyproxy/data-plane-api/pull/539, so this can be updated.

Reply:
data-plane-api commit updated!